build-system/server/amp4test.js

```diff
@@ -95,7 +94,6 @@ app.use('/compose-shadow', function (req, res) {
   const {docUrl} = req.query;
   const viewerHtml = renderShadowViewer({
     src: docUrl.replace(/^\//, ''),
-    port: KARMA_SERVER_PORT,
```

Comment:
Removed intentionally?

Reply:
Yes, port is not used in `renderShadowViewer` and KARMA_SERVER_PORT is not used anywhere other than this call.